README.md

```diff
@@ -227,3 +168,163 @@ Again New Password: ******
 $ henesis logout
 🤗 Logout Success 👍
 ```
+
+
+
+## About henesis.yaml
+
+### webSocket
+
+```yaml
+version: v1
+name: projectname
+
+blockchain:
+  platform: ethereum
+  network: mainnet
+  threshold: 12
+  interval: 1000
+
+filters:
+  contracts:
+    - address: '0x'
+      path: ./contracts/example.sol
+      name: example
+      compilerVersion: 0.5.8
+
+provider:
+  type: webSocket
+```
+
+
+
+### webhook
+
+```yaml
+version: v1
+name: projectname
+
+blockchain:
+  platform: ethereum
+  network: mainnet
+  threshold: 12
+  interval: 1000
+
+filters:
+  contracts:
+    - address: '0x'
+      path: ./contracts/example.sol
+      name: example
+      compilerVersion: 0.5.8
+
+provider:
+  type: webhook
+  url: https://localhost:8080
+  method: POST
+  retry:
+    retryDelay: 1000
+    maxRetries: 3
+  headers:
+    Authorization: 'Bearer YOUR-OWN-TOKEN'
+```
+
+
+
+### parameter details
+
+The following are detailed explanations for parameters used to `henesis.yaml`.
+
+
+
+#### version & name
+
+The `version` and `name` are used as delimiters to identify the project. The `name` must consist only of lowercase letters, numbers, '-' and '.', the maximum length is 253 characters.
+
+
+
+#### blockchain
+
+The blockchain part is an area that describes the platform and network name of the blockchain in which the smart contract to e subscribed is deployed.
+
+##### platform
+
+The blockchain platform you want to use. 
+
+We support now
+
+- `ethereum`
+- `klaytn`
+
+##### network
+
+The blockchain network you want to use.
+
+We support now
+
+- `mainnet` for ethereum and klaytn
+- `ropsten` for ethereum
+- `baobab` for klaytn
+
+##### threshold
+
+Minimum confirmation thresohld which you want to received.
+
+> Caution : when receiving data, we wait for a threshold of block confirmation.
+
+##### interval
+
```

Comment:
interval is no longer used. plz remove it

Reply:
I fixed it at PR#91